src/components/organisms/Navbar/NavbarLink/NavbarLink.tsx

```diff
@@ -133,26 +134,37 @@ const NavbarLink: FC<NavbarLinkProps> = React.forwardRef<HTMLAnchorElement, Navb
       withChevron = false,
       isDropdownLink = false,
       isDropdownHeading = false,
+      onClick,
       ...rest
     },
     ref,
-  ) => (
-    <StyledNavbarLink
-      active={active}
-      withChevron={withChevron}
-      isDropdownLink={isDropdownLink}
-      isDropdownHeading={isDropdownHeading}
-      ref={ref}
-      {...rest}
-    >
-      {withChevron ? <LinkContainer>{children}</LinkContainer> : children}
-      {withChevron && (
-        <ChevronContainer open={open}>
-          <Icon variant={faChevronDown} color={colors.grey} height="12px" width="12px" />
-        </ChevronContainer>
-      )}
-    </StyledNavbarLink>
-  ),
+  ) => {
+    const { setOpen } = useOpen();
+
+    const handleClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
```

Comment:
I think this should go to `<NavbarDropdown/>` into `getItemProps` cause otherwise there is no way to use a custom link

Reply:
Hi @thegrinder I tested this:

```
const SomeLibraryLink = React.forwardRef((props, ref) => {
  const linkProps = Object.assign({}, props);
  delete linkProps.isDropdownLink;


  return <a {...linkProps} onClick={() => console.log('test')} ref={ref} />;
});
```

and it triggered the console. So, seems that the actual fix is working.